core/src/main/java/hudson/tasks/_maven/MavenErrorNote.java

```diff
@@ -52,6 +53,7 @@ public String getDisplayName() {
         }
     }
 
+    @SuppressFBWarnings("MS_SHOULD_BE_FINAL")
```

Comment:
Same?

Reply:
I guess noone will try to modify the pattern with script console, so I'm setting it final.